omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/ConditionController2_2Test.java

```diff
@@ -127,6 +173,7 @@ public void shouldVoidACondition() throws Exception {
 		condition = conditionService.getConditionByUuid(getUuid());
 		Assert.assertTrue(condition.isVoided());
 		Assert.assertEquals("unit test", condition.getVoidReason());
+		Assert.assertNotNull(condition.getDateVoided());
```

Comment:
Can you also assert the voidedBy is equal to the currently authenticated user?

Reply:
Okay